selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/AbstractDeviceTest.java

```diff
@@ -49,10 +49,16 @@ public void testGetCrashLogContents() {
     when(device.getCrashLog()).thenCallRealMethod();
     when(
         device.executeCommandQuietly(argThat(matchesCmdLine("adb shell ls /storage/"))))  // The trailing '/' is key
-        .thenReturn("some_file\nappcrash.log\nanother_file");
+        .thenReturn(String.format("some_file%nappcrash.log%nanother_file"));
+    when(
+        device.executeCommandQuietly(argThat(matchesCmdLine("adb.exe shell ls C:\\storage/"))))  // The trailing '/' is key
```

Comment:
Okay, I see. For some reason I'd assumed it used the system's (Windows, in my case) style. That just leaves the problem with
```
String crashLogDirPath = crashLogFile.getParentFile().getAbsolutePath();
```
because getAbsolutePath will call System.getProperty("user.dir") and return C:\ since the device is being mocked and the System is still Windows. It doesn't seem like you can mock java.lang.System. How would I get around that? Mocking the file class to change the result of getAbsolutePath? Accepting C:\ for the sake of the test? Neither of those seem like particularly nice solutions.